WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -120,4 +124,113 @@ public void onPersonSelected(Person person) {
             fragmentTransaction.commit();
         }
     }
+
+    @Override
+    public void onRoleChanged(long personID, int localTableBlogId, String newRole) {
+        Person person = PeopleTable.getPerson(personID, localTableBlogId);
+        if (person == null || newRole == null || newRole.equalsIgnoreCase(person.getRole())) {
+            return;
+        }
+        PeopleUtils.updateRole(person.getBlogId(), person.getPersonID(), newRole, localTableBlogId,
+                new PeopleUtils.UpdateUserCallback() {
+            @Override
+            public void onSuccess(Person person) {
+                PeopleTable.save(person);
+                refreshOnScreenFragmentDetails();
+            }
+
+            @Override
+            public void onError() {
+                ToastUtils.showToast(PeopleManagementActivity.this,
+                        R.string.error_update_role,
+                        ToastUtils.Duration.LONG);
+            }
+        });
+    }
+
+    private void confirmRemovePerson() {
```

Comment:
It seems that we're not checking for current user when removing. Should that be allowed? If not, perhaps we can put out a message explaining why it cannot be done, or more simply remove the "remove" button completely. Cheers!